tests/unit/util/test_file_converter.py

```diff
@@ -19,6 +19,19 @@ def excel_filepath(sampledir):
     return filepath
 
 
+@pytest.fixture
+def netcdf_filepath(sampledir):
+    netcdf_filepath = os.path.join(sampledir, "aircraft",
+                                   "MOCCA_M251_20190903.nc")
+    return netcdf_filepath
+
+
+@pytest.fixture
+def bad_filepath(sampledir):
```

Comment:
Maybe rename to `csv_filepath`. It's a valid filename, just not one that can (currently) be handled. `bad_filepath`made me expect something totally invalid like "not/a/file.txt".